apps/zotonic_core/src/support/z_context.erl

```diff
@@ -385,6 +386,27 @@ prune_envdata(Env) ->
         cowmachine_resp_body => <<>>
     }.
 
+
+%% @doc Check if the URL is an URL of the local site
+-spec is_site_url( undefined | string() | binary(), z:context() ) -> boolean().
+is_site_url(undefined, _Context) -> false;
+is_site_url(<<"//", _/binary>> = Url, Context) -> is_site_url_1(Url, Context);
```

Comment:
Would a `//` prefixed path ever result in a `true` in `is_site_url_1`? If so, how does it differ from a single `/` prefix?

Reply:
A double prefix is there to use the same protocol as the page. It can contain a different domain name. So if you page is at https://example.com/foo a url "//google.com/foo" will use https as protocol.

Reply:
The `//` is always followed by a hostname, which we then check against the hostnames known by the sites dispatcher.

Reply:
Example:

```
(zotonic@PoToi)4> z_sites_dispatcher:get_site_for_url(<<"//zotonicwww2.test/foo/bar">>). 
{ok,zotonicwww2}
```

Reply:
Aha, thanks for explaining!